src/main/java/com/paneedah/weaponlib/SyncManager.java

```diff
@@ -56,14 +60,14 @@ public void run() {
 	
 	@SuppressWarnings("unchecked")
 	private void sync(PlayerItemInstance<?> watchable) {
-		LOG.debug("Syncing {} in state {} with update id {} to server", watchable, watchable.getState(), watchable.getUpdateId());
+		//LOG.debug("Syncing {} in state {} with update id {} to server", watchable, watchable.getState(), watchable.getUpdateId());
 		long updateId = watchable.getUpdateId(); // capturing update id
 		watchable.setSyncStartTimestamp(System.currentTimeMillis());
 		permitManager.request(new Permit<>((S) watchable.getState()), (PlayerItemInstance<S>)watchable, (p, e) -> {
 			// During sync, the watchable.getUpdateId() can change, so using the original update id
 			watchables.put(watchable, updateId); 
 			watchable.setSyncStartTimestamp(0);
-			LOG.debug("Completed syncing {} with update id {}", watchable, updateId);
+			//LOG.debug("Completed syncing {} with update id {}", watchable, updateId);
```

Comment:
Why? Please revert that, this is good logging

Reply:
In order to be able to see instance.shouldHaveInstanceTags()